ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink

```diff
@@ -182,34 +187,24 @@ is_connected() {
 	fi
 	return 1 # false
 }
+# end function section
```

Comment:
I'm not sure how helpful this comment here is. What about adding a start section comment for the actual code instead?

Reply:
then i also remove the start info here: https://github.com/freifunk-gluon/community-packages/pull/107/files#diff-5bc274003ba90958a75ae0d119e24d89fb0d4acab0bf03ac24797002750e22f3R21

and add something like "# start main logic"?

Reply:
Sure, if you would like that.

Reply:
something like this https://github.com/freifunk-gluon/community-packages/pull/107/commits/c740b1adb5d7f12b842856ebe382831d87caa448 ?

Reply:
Looks fine to me.